tf-explorer.py

```diff
@@ -488,7 +488,7 @@ def help_exit(self):
     print('Syntax: exit')
 
   def do_exit(self, arg):
-    self.do_EOF(self, arg)
+    self.do_EOF(arg)
```

Comment:
Can you add a return statement here so it's `return self.do_EOF(arg)`? Otherwise the exit command won't do anything.